src/loki/worker.cc

```diff
@@ -142,6 +142,19 @@ void loki_worker_t::parse_costing(Api& api, bool allow_none) {
     }
   }
 
+  if (options.has_chinese_polygon()) {
+    const auto chinese_edges =
+        edges_in_ring(options.chinese_polygon(), *reader, costing, max_exclude_polygons_length);
```

Comment:
I'm not sure if piggy backing on `max_exclude_polygons_length` is good.. maybe it is, instead of introducing yet another, almost identical config option? both are very expensive, though chinese postman much more so..

Reply:
Ah yes, you are right. I forgot to add a new one. Added now as max_chinese_polygon_length